cicecore/drivers/nuopc/cmeps/ice_comp_nuopc.F90

```diff
@@ -1331,51 +1329,51 @@ subroutine ModelSetRunClock(gcomp, rc)
        call ESMF_LogWrite(subname//'setting alarms for ' // trim(name), ESMF_LOGMSG_INFO)
 
        !----------------
-       ! Restart alarm
+       ! Stop alarm
```

Comment:
Could you re-order the ``Stop alarm`` and ``Restart alarm`` code blocks? I believe that will remove this file from having any differences.

Reply:
No - the reason they are reordered is so that a REST_OPTION=end can be initialized using the stop alarm.   If you do the rest alarm first you can not initialize this option.

Reply:
See line 927 of ice_shr_methods.F90